index.css

```diff
@@ -1,55 +1,58 @@
-@tailwind base;
+@import 'tailwindcss';
 
-:root {
-  --c-text: #333;
-  --c-background: #fff;
-}
+@theme {
```

Comment:
Theme extensions that used to be in `tailwind.config.js` now just live under the `@theme` layer directive. Native CSS for the win!